packages/core/src/lib/list-box/__tests__/list-box.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { describe, it, expect } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/svelte';
+import { ListBox } from '$lib';
+import { default as ListBoxSlot } from '../list-box-slot.svelte';
+
+const defaultProps = {
+  disabled: false,
+  left: 'leftText1,leftText2,leftText3',
+  right: 'rightText1,rightText2,rightText3',
+  leftlabel: 'Left',
+  rightlabel: 'Right',
+  height: '200px',
+  suffix: false,
+};
+
+describe('ListBox', () => {
+  it('Renders left options and can move them to the right', async () => {
+    render(ListBoxSlot, { ...defaultProps });
+
+    await fireEvent.click(screen.getByText('leftText1'));
```

Comment:
Cousins recently gave me some feedback in one of my other prime core migration PRs saying that it's better to use user event instead of fireEvent based on the testing library documentation.  

https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent

Reply:
👍 swapped it all over